docs/installation/update.rst

```diff
@@ -22,19 +32,21 @@ Upgrade Steps
 
     conda activate tethys
 
-3. Update the Tethys Platform conda package and dependencies by installing the new version of Tethys Platform::
+3. Update the Tethys Platform conda package and dependencies by installing the new version of Tethys Platform:
 
-    conda install -c tethysplatform -c conda-forge tethys-platform=3.*
+    .. code-block:: bash
 
-4. Migrate the Tethys Platform database tables::
+        conda install -c conda-forge -c tethysplatform tethys-platform=4.*
```

Comment:
Can we add `tethys-platform` to conda-forge now?

Reply:
That would be the goal I think. Though the tethys_dataset_services package hasn't shown up yet.